<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="combo_Video">
          <item>
           <property name="text">
            <string>Camera Video0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Camera Video1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Camera Video2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>File Video</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Open">
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_CLose">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Start">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_Stop">
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>778</width>
           <height>522</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Open</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_OpenVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_CLose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_CloseVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_StartVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_button_StopVideo_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_Video</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_combo_Video_Change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_button_OpenVideo_clicked()</slot>
  <slot>on_button_CloseVideo_clicked()</slot>
  <slot>on_button_StartVideo_clicked()</slot>
  <slot>on_button_StopVideo_clicked()</slot>
  <slot>on_combo_Video_Change(QString)</slot>
 </slots>
</ui>
